### PR TITLE
feat: makefile

### PR DESCRIPTION
**Pull request by Glangouet**
Install Akeneo project with Makefile

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+docker-shared: docker-network docker-consul docker-registrator docker-fabio docker-ssh-agent docker-mysql
+
+## Create a docker network
+docker-network:
+	[ -n "$(shell docker network ls -q --filter name='^$(DOCKER_NETWORK)$$' 2>/dev/null)" ] \
+	 || { echo -n "Creating docker network $(DOCKER_NETWORK) ... " && docker network create $(DOCKER_NETWORK) >/dev/null 2>&1 && echo "done" || echo "ERROR"; }
+
+## Generate a SSL certificate for $(DOCKER_LOCAL_HOSTNAME)
+docker-openssl: /etc/ssl/certs/$(DOCKER_LOCAL_HOSTNAME).crt /etc/ssl/certs/$(DOCKER_LOCAL_HOSTNAME).key
+# letsencrypt wont work yet as local.1001pharmacies.com is not a valid DNS A record
```
Comment (crazyball) [inline Makefile]: commentaires a supprimer

File: .gitignore [hunk 2]
```diff
@@ -15,6 +15,7 @@ app/uploads/product/*
 app/file_storage/*
 app/bootstrap.php.cache
 build/
+pim-project
```
Comment (iGitScor) [inline .gitignore]: What does this folder contain ?

File: README.md [hunk 3]
```diff
@@ -1,51 +1,11 @@
-Akeneo PIM Community Standard Edition
```
Comment (iGitScor) [inline README.md]: If you remove all those content, maybe add a link to Akeneo documentation (or repository)

File: docker-compose.yml.dist [hunk 4]
```diff
@@ -3,46 +3,30 @@ version: '2'
...
-      PHP_IDE_CONFIG: 'serverName=pim-ce'
-      PHP_XDEBUG_ENABLED: 0
-      PHP_XDEBUG_IDE_KEY: XDEBUG_IDE_KEY
-      PHP_XDEBUG_REMOTE_HOST: xxx.xxx.xxx.xxx
-      XDEBUG_CONFIG: 'remote_host=xxx.xxx.xxx.xxx'
+      COMPOSER_HOME: '/root/.composer'
+      SSH_AUTH_SOCK: /tmp/ssh-agent/socket
+    labels:
+        - SERVICE_80_NAME=akeneo
+        - SERVICE_80_CHECK_HTTP=/
```
Comment (iGitScor) [inline docker-compose.yml.dist]: Same as here https://github.com/1001Pharmacies/mautic/pull/2#discussion_r135704759

**Pull request closed without merging**